lz4/go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/gocql/gocql/lz4
 go 1.16
 
 require (
-	github.com/pierrec/lz4/v4 v4.1.8
+	github.com/pierrec/lz4/v4 v4.1.14
```

Comment:
> Please either update the commit message or remove the version bump.

Done

Reply:
Thanks! Now I see that the commit message mentions v4.1.17, but there is still v4.1.14 in go.mod. Please change the version in go.mod to 4.1.17 so that it matches the commit message.

Reply:
> Thanks! Now I see that the commit message mentions v4.1.17, but there is still v4.1.14 in go.mod. Please change the version in go.mod to 4.1.17 so that it matches the commit message.

Gah, sorry; I got some wires crossed there it seems.